src/overlays/actors/ovl_Bg_Mori_Hineri/z_bg_mori_hineri.c

```diff
@@ -7,8 +13,17 @@
 void BgMoriHineri_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgMoriHineri_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgMoriHineri_Update(Actor* thisx, GlobalContext* globalCtx);
+void func_808A3F58(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
imo, a line break between these two to make ti clear that its seperate from the main funcs that get set in the init vars
Also, I looked in the game to see what this is and it draws the whole hallway and the room (except for the door and the chute)
So BgMoriHineri_DrawHallAndRoom is probably ok

Reply:
I think keeping them together is much better. There is no difference between them in terms of their content/purpose.

Reply:
I like seeing from a glance which were "additional" from the usual 4, but I do understand that they serve the same function and shouldn't be separated for that reason. disregard my comment lucas